pystac_monty/sources/glide.py

```diff
@@ -62,7 +94,11 @@ def make_source_event_items(self) -> List[Item]:
                 glide_id = STAC_EVENT_ID_PREFIX + data.get("event") + "-" + data.get("number") + "-" + data.get("geocode")
                 latitude = float(data.get("latitude"))
                 longitude = float(data.get("longitude"))
-                event_date = {"year": data.get("year"), "month": data.get("month"), "day": data.get("day")}
+                event_date = {
+                    "year": abs(data.get("year")),
+                    "month": abs(data.get("month")),
+                    "day": abs(data.get("day")),
```

Comment:
are you sure we get the values in int type? Also, better to typecast to int as well.